### PR TITLE
Allow to simply run a script once - no matter what the changes are

### PR DESCRIPTION
**Pull request by laurentsigal**

### CODE REVIEW

File: pre_commit/clientlib/validate_manifest.py [hunk 1]
```diff
@@ -17,6 +17,7 @@ class InvalidManifestError(ValueError):
         'type': 'object',
         'properties': {
             'id': {'type': 'string'},
+            'always_run': {'type': 'boolean'},
```
Comment (asottile) [inline pre_commit/clientlib/validate_manifest.py]: we can `'default': False` here and then the check for this gets simpler in pre_commit/run.py

File: pre_commit/commands/run.py [hunk 2]
```diff
@@ -72,13 +72,19 @@ def get_filenames(args, include_expr, exclude_expr):
 
 
 def _run_single_hook(hook, repo, args, write, skips=frozenset()):
-    filenames = get_filenames(args, hook['files'], hook['exclude'])
+    filenames = []
+    # if the hook is marked as always_run, do not compute the files to run
+    # in that case, simply run the script once not matter the changes
+    compute_file_names = 'always_run' not in hook or not hook['always_run']
```
Comment (asottile) [inline pre_commit/commands/run.py]: I almost want this to be less special and still compute the filenames.  That'd be more in line with the rest of the project and I think make things simpler to reason about.

File: pre_commit/clientlib/validate_manifest.py (hunk 1 above)
Comment (laurentsigal) [inline pre_commit/clientlib/validate_manifest.py]: yeah good point!

File: pre_commit/commands/run.py (hunk 2 above)
Comment (laurentsigal) [inline pre_commit/commands/run.py]: sounds good

**Comment by laurentsigal** [comment]
That's certainly cleaner

**Comment by laurentsigal** [comment]
Fixing the tests

**Comment by asottile** [comment]
@laurentsigal sweet! I wrote a test for you and PR'd your branch here: https://github.com/laurentsigal/pre-commit/pull/1

**Comment by laurentsigal** [comment]
@asottile dude you're super efficient!

**Comment by asottile** [comment]
Alright, this is released in version 0.7.2!

The last things to do are:
- Fork this to add yourself (if you'd like) and document the new feature: https://github.com/pre-commit/pre-commit.github.io

Thanks for the PR!

**Comment by asottile** [comment]
@laurentsigal  I've updated the docs for your change! If you like to add yourself to the contributors on http://pre-commit.com feel free to PR https://github.com/pre-commit/pre-commit.github.io

**Comment by laurentsigal** [comment]
@asottile Sorry I was on a Christmas break. Thanks a lot for the help on adding this to pre-commit! 
I just added myself to the list of contributors https://github.com/pre-commit/pre-commit.github.io/pull/71

**Pull request merged**